cekit/tools.py

```diff
@@ -608,6 +614,10 @@ def handle_core_dependencies(self) -> None:
         try:
             import certifi
 
+            # If the certificate bundle ends up resolving e.g. /usr/local/lib/python3.7/site-packages/certifi/cacert.pem
+            # rather than /etc/pki/tls/certs/ca-bundle.crt or /etc/ssl/certs/ca-bundle.crt this can cause issues
+            # hence printing the warning. A user workaround is to export REQUESTS_CA_BUNDLE to point to the correct
+            # certificates
```

Comment:
I wonder if we could add this information to the warning message below as well.

Reply:
This, or similar, that is.

Reply:
Added another log statement.